pkgs/by-name/mi/minutor/package.nix

```diff
@@ -0,0 +1,43 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  qt5,
+  zlib,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "minutor";
+  version = "2.21.0";
+
+  src = fetchFromGitHub {
+    owner = "mrkite";
+    repo = finalAttrs.pname;
+    rev = "ref/tags/${finalAttrs.version}";
+    sha256 = "0ldjnrk429ywf8cxdpjkam5k73s6fq7lvksandfn3xn7gl9np5rk";
+  };
+
+  preConfigure = ''
```

Comment:
```suggestion
  postPatch = ''
```

Reply:
Logically belongs into `configurePhase` since we set the installation location here (as the application lacks any sort of configure script, sadly). Having stuff that depends on `$out` in `patchPhase` doesn't make sense, e.g. just think about `srcOnly` which would then reference the store path of the resulting patched source.